pkg/storage/emptydir/emptydir.go

```diff
@@ -73,6 +73,9 @@ func (d *driver) StorageChanged(cr *imageregistryv1.Config) bool {
 }
 
 func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
+	if cr.Spec.Storage.ManagementState == "" {
+		cr.Spec.Storage.ManagementState = imageregistryv1.StorageManagementStateUnmanaged
```

Comment:
Maybe it's better to say that emptyDir is also managed by default?

Reply:
done.